route/pool.go

```diff
@@ -43,7 +43,8 @@ func NewStats() *Stats {
 
 type Endpoint struct {
 	ApplicationId        string
-	addr                 string
+	Address              string
```

Comment:
This change looks pretty innocuous but it's also in the original non-F5 code. Was there a specific reason it was changed? At the very least these values should remain un-exported.

Reply:
No reason to change it. I thought I needed to export it, but I don't, so I will change it back.

Reply:
+1

Reply:
Actually, I do export it now. Pool members are objects now (were a "addr:port" strings before), So I need to export the Address and Port of the Endpoint to create the pool members.

Reply:
Ok, sounds reasonable.